### PR TITLE
Add getPartnerInventory method

### PR DESCRIPTION
**Pull request by Luc4sguilherme**
I am adding this method as it is not affected by the new rate limit imposed by steam in the last update.

All other methods of obtaining an inventory have been affected and this is the only decent way to obtain one now.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -276,6 +277,80 @@ TradeOfferManager.prototype._getFromDisk = function(filenames, callback) {
...
 
+/**
+ * Get the contents of a partner's inventory.
+ * @param {SteamID|string} partnerID - The partner's SteamID as a SteamID object or a string which can parse into one
+ * @param {int} appID - The Steam application ID of the game for which you want an inventory
+ * @param {int} contextID - The ID of the "context" within the game you want to retrieve
+ * @param {boolean} tradableOnly - true to get only tradable items and currencies
+ * @param {function} callback
+ */
+ TradeOfferManager.prototype.getPartnerInventory = function(partnerID, appID, contextID, tradableOnly, callback) {
```
[the remark this answers is not in the document]
Comment (Luc4sguilherme) [reply lib/index.js]: I really only get tradable items regardless of the value of this parameter.

File: lib/index.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ const EConfirmationMethod = TradeOfferManager.EConfirmationMethod = require('../
 const ETradeStatus = TradeOfferManager.ETradeStatus = require('../resources/ETradeStatus.js');
 
 const TradeOffer = require('./classes/TradeOffer.js');
+const CEconItem = require('steamcommunity/classes/CEconItem.js');
```
[the remark this answers is not in the document]
Comment (Luc4sguilherme) [reply lib/index.js:22]: @DoctorMcKay Could I do it this way, or can I return `CEconItem`?

```js
inventory.push(new EconItem(new CEconItem(body.rgInventory[i], body.rgDescriptions, contextID)));
```

[the remark this answers is not in the document]
Comment (Luc4sguilherme) [reply lib/index.js:22]: When I do this the boolean values is not handled correctly.

[the remark this answers is not in the document]
Comment (Luc4sguilherme) [reply lib/index.js:22]: But I think I would have no problem returning CEconItem, as there are other methods in this package that do that.

File: lib/index.js [hunk 3]
```diff
@@ -305,7 +305,6 @@ TradeOfferManager.prototype._getFromDisk = function(filenames, callback) {
 				"appid": appID,
 				"contextid": contextID,
 				"start": start,
-				"trading": 1,
```
Comment (Luc4sguilherme) [inline lib/index.js]: This parameter doesn't seem to have any effect, so it seems to be useless. Steam uses this parameter only when it is loading its own inventory. So it's better to remove it to have the same behavior.

**Comment by Luc4sguilherme** [comment]
Steam appears to have made a new update that has applied new rate limits to this method. So it looks like it's no longer useful.

**Comment by nolddor** [comment]
Big F

**Pull request closed without merging**